src/main/java/racingcar/view/ResultView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.view;
+
+import racingcar.Cars;
+
+public class ResultView {
+
+    private static final String MOVED_EXPRESSION = "-";
+
+    private ResultView() {
+        throw new IllegalStateException("Utility class.");
+    }
+
+    public static void printCarsPosition (Cars cars) {
```

Comment:
```suggestion
    public static void printCarsPosition(Cars cars) {
```

스트림을 활용해주셨네요 👍 

객체지향 생활 체조 원칙 **규칙 4: 한 줄에 점을 하나만 찍는다.**  에 맞게 수정해볼까요?